misk-aws/src/main/kotlin/misk/jobqueue/sqs/AwsSqsJobQueueModule.kt

```diff
@@ -72,9 +73,12 @@ class AwsSqsJobQueueModule(
   ): RepeatedTaskQueue {
     return queueFactory.new(
         "sqs-consumer-poller",
-        config.task_queue)
+        repeatedTaskQueueConfig(config))
   }
 
+  private fun repeatedTaskQueueConfig(config: AwsSqsJobQueueConfig) =
+      config.task_queue ?: RepeatedTaskQueueConfig(num_parallel_tasks = -1)
```

Comment:
i don't see value in allowing somebody to configure a bounded task queue for this. can we just remove the config entirely from the aws job queue config?

Reply:
neither do I. Yes, I can remove it. I just want to make sure I don't break existing clients.

Reply:
i would just grep for existing clients. at the very least if you don't remove, mark it clearly deprecated with instructions to remove it so you don't have a single threaded consuming app!